doc/customizations-configure-external-postgres.md

```diff
@@ -0,0 +1,154 @@
+### External database
```

Comment:
Can be removed, except the Azure part that we need to add to the doc. Could you add the azure data to the current doc PR?

Reply:
Done, added here - https://app.gitbook.com/o/zP3LT2Qhlb5jPWQCfWV3/s/ybPdpmLltPkzGFvz7m8A/~/changes/467/carto-self-hosted/deployment-guides/single-vm-deployment#configure-the-external-database